packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -584,7 +585,28 @@ export function clearSuspenseBoundaryFromContainer(
   retryIfBlockedOn(container);
 }
 
+function instanceContainsElem(instance: Instance, element: HTMLElement) {
+  let fiber = getClosestInstanceFromNode(element);
+  while (fiber !== null) {
+    if (fiber.tag === HostComponent && fiber.stateNode === element) {
```

Comment:
It might be a bit strange that we expose Fiber details in the host config, but we do this in other places.